src/main/java/network/brightspots/rcv/GuiTiebreakerController.java

```diff
@@ -28,6 +28,7 @@
 public class GuiTiebreakerController {
 
   private String candidateToEliminate = null;
+  private boolean tabulationCancelled = false;
```

Comment:
Super nitpicky, but American English favors canceled/canceling over cancelled/cancelling. Whose side are you on?

Reply:
I hadn't thought about this too much, but I'm on the side of "cancelled" because "cancellation" is double-l no matter which version of English you use.